templates/footer.hbs

```diff
@@ -1,22 +1,175 @@
3-1.695 1.9-2.726a9.388 9.388 0 00.567-3.255c0-2.754-1.05-4.818-3.147-6.194zM168.58 27.229c-.485.461-1.417.692-2.793.692h-2.896a.912.912 0 01-.91-.914v-3.503c0-.505.407-.914.91-.914h2.896c1.376 0 2.308.231 2.793.692.487.461.73 1.126.73 1.993 0 .841-.243 1.493-.73 1.954zm-6.599-14.181c0-.505.407-.914.91-.914h2.692c1.135 0 1.904.19 2.31.57.404.38.606.922.606 1.626 0 .707-.202 1.249-.606 1.628-.406.38-1.175.57-2.31.57h-2.692a.912.912 0 01-.91-.914v-2.566zm12.847 6.766a.91.91 0 01-.074-1.468c.672-.54 1.193-1.153 1.564-1.839.564-1.044.847-2.23.847-3.56 0-1.003-.162-1.96-.486-2.868-.323-.909-.868-1.702-1.636-2.38-.768-.678-1.8-1.214-3.092-1.607-1.295-.394-2.912-.59-4.851-.59h-12.591c-.502 0-.91.409-.91.913v27.223c0 .505.408.914.91.914h12.228c2.101 0 3.866-.23 5.295-.692 1.428-.46 2.587-1.084 3.476-1.871.89-.786 1.523-1.695 1.9-2.726a9.388 9.388 0 00.567-3.255c0-2.754-1.05-4.818-3.147-6.194zM207.219 27.432h-11.37a.912.912 0 01-.908-.914v-2.16c0-.504.407-.913.909-.913h9.842c.503 0 .91-.41.91-.914v-5.293a.912.912 0 00-.91-.914h-9.842a.912.912 0 01-.91-.914v-1.874c0-.505.408-.914.91-.914h10.652c.502 0 .91-.41.91-.914V6.415a.912.912 0 00-.91-.914H186.82c-.502 0-.91.41-.91.914v27.223c0 .505.408.914.91.914h19.886c.468 0 .86-.357.905-.825l.512-5.292a.912.912 0 00-.905-1.003M230.759 17.748c-.576.516-1.55.774-2.919.774h-2.148a.912.912 0 01-.91-.914V13.25c0-.505.408-.914.91-.914h2.148c1.37 0 2.343.258 2.919.773.578.516.866 1.289.866 2.32 0 1.03-.288 1.803-.866 2.318m10.547 15.456l-4.884-9.086a.909.909 0 01.245-1.154 8.995 8.995 0 002.51-2.937c.742-1.384 1.113-3.052 1.113-5.005a11.54 11.54 0 00-.588-3.702 7.239 7.239 0 00-1.967-3.032c-.92-.854-2.129-1.531-3.63-2.034-1.5-.502-3.36-.753-5.578-.753h-11.5c-.502 0-.91.41-.91.914v27.223c0 .505.408.914.91.914h6.846c.502 0 .91-.409.91-.914v-7.49c0-.504.407-.913.909-.913h1.765a.91.91 0 01.815.506l4.115 8.305c.154.31.469.506.814.506h7.305a.914.914 0 00.8-1.348M36.97 25.379c0 6.5-5.244 11.77-11.713 11.77H8.65a.382.382 0 01-.381-.383v-22.01c0-3.56 2.872-6.448 6.416-6.448h11.141c.266 0 .45.267.358.517l-.648 1.75a.382.382 0 01-.357.25h-9.954c-2.46 0-4.453 2.003-4.453 4.474V34.25c0 .212.171.384.382.384h14.103c5.086 0 9.21-4.144 9.21-9.255V5.896c0-.342.41-.513.651-.271l1.74 1.749c.072.072.113.17.113.27V25.38zm-23.404.74V15.3c0-.92.743-1.667 1.66-1.667h10.768c.21 0 .381.172.381.383v10.821c0 .92-.742 1.667-1.659 1.667H13.948a.383.383 0 01-.382-.383zM5.475 14.757v19.483a.381.381 0 01-.652.271l-1.74-1.749a.385.385 0 01-.112-.271V14.756c0-6.5 5.244-11.77 11.714-11.77H31.29c.211 0 .382.17.382.382v22.01c0 3.562-2.873 6.448-6.416 6.448H14.115a.383.383 0 01-.357-.517l.647-1.749a.382.382 0 01.358-.25h9.953c2.46 0 4.453-2.003 4.453-4.474V5.884a.382.382 0 00-.381-.383H14.685c-5.087 0-9.21 4.143-9.21 9.255zM33.984.515a1.142 1.142 0 00-.81-.337h-18.49C6.673.178.178 6.705.178 14.756v18.579c0 .305.12.597.335.813l5.446 5.472c.214.216.506.337.81.337h18.489c8.012 0 14.508-6.527 14.508-14.578V6.799c0-.304-.121-.597-.336-.812L33.984.515z"/></g></svg>
```

Comment:
Can you include this SVG in another way? can we have like a `_logo.hbs` file that gets included? That would make the logo itself more reusable.

Reply:
Nope. I did try it but it looks like SVG still has the limitation that is needs to be inline if you want to style it.